sqlglot/generator.py

```diff
@@ -97,6 +97,7 @@ class Generator:
         exp.SampleProperty: lambda self, e: f"SAMPLE BY {self.sql(e, 'this')}",
         exp.SetProperty: lambda self, e: f"{'MULTI' if e.args.get('multi') else ''}SET",
         exp.SettingsProperty: lambda self, e: f"SETTINGS{self.seg('')}{(self.expressions(e))}",
+        exp.SqlReadWriteProperty: lambda self, e: self.sql(e.this),
```

Comment:
Nit: can also do this (it matches what we do for `StabilityProperty`):
```suggestion
        exp.SqlReadWriteProperty: lambda self, e: e.name,
```